tap_quickbooks/schemas/bills.json

```diff
@@ -229,13 +210,125 @@
       "format": "singer.decimal"
     },
     "DepartmentRef": {
-      "$ref": "ref_schema.json"
+      "$ref": "shared/ref_schema.json"
     },
     "PrivateNote": {
       "type": [
         "null",
         "string"
       ]
+    },
+    "HomeBalance": {
+      "format": "singer.decimal",
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "IncludeInAnnualTPAR": {
+      "type": [
+        "null",
+        "boolean"
+      ]
+    },
+    "TransactionLocationType": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "RecurDataRef": {
+      "$ref": "shared/ref_schema.json"
+    },
+    "TxnTaxDetail": {
```

Comment:
Can we use shared/txn_tax_detail.json ref here?

Reply:
Updated.